docs/dictionary/control_st/return.lcdoc

```diff
@@ -2,39 +2,142 @@ Name: return
 
 Type: control structure
 
-Syntax: return <value> 
+Syntax: return [ value | error ] <value>
```

Comment:
Aha!... my syntax checker is broken because:
```
not "return [ value | error ] <value>" contains "port" is false 
```
while
```
not ("return [ value | error ] <value>" contains "port") is true
```
A mistake I should not have made after so long scripting in LC!